src/options.ts

```diff
@@ -1,19 +1,22 @@
-/** @type {Record<string, import('prettier').SupportOption>} */
-export const options = {
+import type { RequiredOptions, SupportOption } from 'prettier'
+import type { Customizations } from './types'
+import './index'
+
+export const options: Record<string, SupportOption> = {
   tailwindConfig: {
-    since: '0.0.0',
+    // since: '0.0.0',
```

Comment:
Yep. Turns out the `since` key isn't valid (maybe never was?) in Prettier options. So I commented them out. I've removed the comments to eliminate any confusion around this.